src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,114 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import duke.command.AddCommand;
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.DueCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+
+public class Parser {
+    private static final String TODO = "todo";
+    private static final String DELETE = "delete";
+    private static final String DONE = "done";
+    private static final String EVENT = "event";
+    private static final String DEADLINE = "deadline";
+    private static final String LIST = "list";
+    private static final String DUE = "due";
+    private static final String FIND = "find";
```

Comment:
I like how the static variables which are used to keep track of commands improve the code readability. Perhaps you could consider wrapping all these static variables in a static class to make it a bit more organised.

Something like:
```
private static class CommandTypes
{
    public static final String TODO = "todo";
    public static final String DELETE = "delete";
}
```